tests/test_art.py

```
@@ -0,0 +1,77 @@
+        with patch.dict(os.environ, {
+            "OPENAI_ENABLED": "True",
+            "OPENAI_KEY": "test-key"
+        }):
+            mock_response = Mock()
+            mock_response.data = [Mock()]
+            mock_response.data[0].url = "https://example.com/image.png"
+
+            with patch('src.art.openai_client.images.generate', new=AsyncMock(return_value=mock_response)):
+                result = await draw("openai", "A beautiful sunset")
+
+                assert result == "https://example.com/image.png"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)